@xen-orchestra/lite/src/views/vm/VmConsoleView.vue

```diff
@@ -1,20 +1,34 @@
 <template>
   <div v-if="!isReady">Loading...</div>
   <div v-else-if="!isVmRunning">Console is only available for running VMs.</div>
-  <RemoteConsole
-    v-else-if="vm && vmConsole"
-    :location="vmConsole.location"
-    :is-console-available="!isOperationsPending(vm, STOP_OPERATIONS)"
-  />
+  <template v-else-if="vm && vmConsole">
+    <RemoteConsole
+      :is-console-available="!isOperationsPending(vm, STOP_OPERATIONS)"
+      :location="vmConsole.location"
+      class="remote-console"
+    />
+    <RouterLink
+      v-if="uiStore.hasUi"
+      :to="{ query: { ui: '0' } }"
+      class="open-link"
+      target="_blank"
+    >
```

Comment:
I think it should open in a popup window ideally with the right dimensions of the current console.